stories/tables/sortable-table.story.js

```diff
@@ -91,6 +91,13 @@ storiesOf('SortableTable', module)
       <Column name="active" component={CustomCell} />
     </SortableTable>
   ))
+  .add('with custom properties on cell component per column', () => (
+    <SortableTable data={tableData}>
+      <Column name="name" customCellClass="test" />
```

Comment:
This is a bit misleading since `customCellClass` will be filtered out on the default column component. I think you could add `customCellClass` to the story book example "with custom cell component" instead